src/pages/Task.tsx

```diff
@@ -13,15 +13,22 @@ import {
   PageLayout,
   PageSider
 } from "components/styles";
+import { TestsTable } from "./task/TestsTable";
+import { FilesTables } from "./task/FilesTables";
 
-enum Tab {
-  Logs = "logs",
-  Tests = "tests",
-  Files = "files",
-  BuildBaron = "build-baron"
-}
 const DEFAULT_TAB = Tab.Logs;
 
+const getBodyComp = tab => {
```

Comment:
render functions are typically named beginning with render. Does not matter much because this will be replaced in my PR with the Tabs logic. But `renderTab` would be a much clearer name imo.